### ISSUE TITLE
How to use bonn dataset gt pose?

### ISSUE THREAD

**Issue by jinjiwu** [PRBonn/refusion#14]
When I use ground truth (GT) poses from the dataset to reconstruct the point cloud (balloon), the point cloud doesn't align well. However, when I use the TUM RGB-D dataset (fr3_walking_half), it aligns well. Did I do something wrong?

![snapshot01](https://github.com/PRBonn/refusion/assets/24726424/f3977ea4-b050-4b5f-b005-e28f686f0bb4)

Really hope to get your help!

**Comment by jbehley** [comment]
Hi @jinjiwu, 

sorry for the late reply. @Eman7C7 left our lab already some time ago and probably hasn't time anymore to look over each issue. But maybe I try to help you.

Have you used the instructions on the webpage:  https://www.ipb.uni-bonn.de/index.html%3Fp=2544.html

**Comment by jinjiwu** [comment]
Yes. I'm choosing the depth image and unprojecting it to a point cloud using the GT pose from groundtruth.txt to obtain the complete point cloud. I'm selecting the GT pose with the timestamp that's closest to the depth timestamp. I believe the issue arises from a misalignment between the GT timestamp and the depth image

**Comment by jinjiwu** [comment]
Sorry to bother you again, but could you please provide some advice on how to solve this problem?
